lumeer-storage/lumeer-storage-api/src/main/java/io/lumeer/storage/api/dao/GroupDao.java

```diff
@@ -24,12 +24,12 @@
 
 public interface GroupDao {
 
-   String createGroup(Group group);
+   Group createGroup(String organizationId, Group group);
 
-   void updateGroup(String id, Group group);
+   Group updateGroup(String id, Group group);
 
    void deleteGroup(String id);
 
-   List<Group> getAllGroups();
+   List<Group> getAllGroups(String organizationId);
```

Comment:
Once you fix the problem with the scope of this DAO, `organizationId` parameter will not be needed in any of these methods.